sdk/communication/azure-communication-rooms/setup.py

```diff
@@ -65,7 +65,7 @@
         ]
     ),
     install_requires=[
-        "azure-core>=1.24.0",
+        "azure-core>=1.29.0",
         "msrest>=0.7.1",
```

Comment:
msrest should not be a required dependency, please update:
https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/communication/azure-communication-chat/setup.py#L66-L69